modules/gatsby/src/components/query-layer.jsx

```diff
@@ -0,0 +1,99 @@
+// NOTE: Dummy component, just to get gatsby to compile the StaticQuery
+
```

Comment:
Add more comment, here explaining that this must be identical to the query in ocular-gatsby and must not be modified.